infrateam_first_responder.md

```diff
@@ -15,9 +15,61 @@ It can be helpful to scan the list of PRs generated in Slack.  If you notice som
 
 #### Deploy 'em
 
-Use the `sdr-deploy` CLI, from your laptop, to deploy all infrastructure projects via capistrano to deployed environments: https://github.com/sul-dlss/sdr-deploy.
+Use the `sdr-deploy` CLI, from the sdr-infra server, to deploy all infrastructure projects via capistrano to deployed environments: https://github.com/sul-dlss/sdr-deploy.
 
-Note that you will need to be sure you can ssh into each of the VMs from your laptop. (See the [sdr-deploy README](https://github.com/sul-dlss/sdr-deploy/blob/main/README.md) for more about how to use the `check_ssh` command to do this.)
+Note that you will need to be sure you can ssh into each of the VMs from sdr-infra. (See the [sdr-deploy README](https://github.com/sul-dlss/sdr-deploy/blob/main/README.md) for more about how to use the `check_ssh` command to do this.)
+
+##### 1. Setup sdr-infra for deploying
+
+###### Login with Multi-factor Authentication (MFA)
+
+From your laptop:
+```
+ssh -t sdr-infra.stanford.edu
+```
+Note: the `-t` is important here to forward your ssh agent key(s) Once logged in you can verify your key(s) are available with `ssh-add -L` and compare to your local output.
```

Comment:
I wonder how folks would feel about documenting putting this in SSH config instead of as a command-line flag, like what's recommended in https://docs.github.com/en/authentication/connecting-to-github-with-ssh/using-ssh-agent-forwarding. So there could be a step 0 in here about making sure you have SSH agent forwarding set up for Stanford VMs in which case we might recommend config like:

```
## Uncomment these so you can SSH to boxes without the full .stanford.edu domain name
##   e.g., type `ssh sdr-infra` and the SSH client will use `*.stanford.edu` config
# CanonicalizeHostname yes
# CanonicalDomains stanford.edu

Host *.stanford.edu
    # Forward your SSH key so you can interact with GitHub
    ForwardAgent yes  
    # Set up Kerberos authentication
    GSSAPIAuthentication yes
    GSSAPIClientIdentity YOURSUNETIDHERE@stanford.edu
    GSSAPIDelegateCredentials yes
    GSSAPIKeyExchange yes
    # Specify your Stanford username here
    User YOURSUNETIDHERE
```